medical_lab/models/medical_lab_test_criterion.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLabTestCriterion(models.Model):
+
```

Comment:
You should just always put a blank line really. PEP-008 is king; I'm not sure why I make exceptions sometimes. Probably because nobody calls me out on it /shrug